resource-watch/resource-watch
# Legend

## Overview
This PR refactor legend

## Testing instructions
Go to this places to be sure that everything works
- components/admin/layers/form/steps/Step1.js
- components/dashboards/wysiwyg/widget-block/widget-block-component.js
- components/widgets/list/WidgetCard.js
- pages/app/embed/EmbedLayer.js (as we are going to refactor the Explore Page we should make a better embed of it. Also, we are not using this embed anywhere...)
- pages/app/embed/EmbedMap.js
- pages/app/Explore.js
